endpoints/getting-started-grpc/README.md

```diff
@@ -77,7 +77,6 @@ Cloud account and [SDK](https://cloud.google.com/sdk/) configured.
     ```bash
```

Comment:
Up above:
- Change "gcloud service-management deploy ..." -> "gcloud endpoints services deploy ..."
- Remove the comment directly underneath about noting the configuration ID.
- Remove the command to print out the configuration ID.